rfcs/interop_2023.md

```diff
@@ -0,0 +1,84 @@
+# RFC 116: Interop 2023
+
+## Summary
+
+Interop 2023 is an effort to improve interoperability in areas believed to be important to web developers and users. This is the next iteration of [Interop 2022](https://github.com/web-platform-tests/interop-2022), an effort which is still ongoing at the time of this RFC.
+
+The focus areas will be selected by a proposals period followed by a proposals review and selection process. The [Interop 2023 team](https://github.com/orgs/web-platform-tests/teams/interop-2023) will manage this process and make decisions based on consensus, see [governance](#governance).
+
+There will be a dashboard at https://wpt.fyi/interop-2023.
+
+## Details
+
+### Timeline
+
+During 2022:
+
+- September 15 to October 15: Proposals period.
+- October 16 to October 30: Proposal discussion/refinement. (New proposals no longer accepted.)
+- November 1 to November 30: Proposal selection. (The team must reach consensus no later than November 30.)
+- December 1 to December 16:
+  - Start review of tests. Write needed tests.
+  - Start involving PR teams. Write rough drafts of announcements. Begin creating dashboard.
+
+During 2023:
+
+- January 9 to January 20:
+  - Finalize test selection.
+  - Complete announcement plans. Prepare dashboard for launch.
+- January 23 to January 27: Launch.
+
+If necessary, additional changes to the test lists can be made throughout the year with the consensus of the team, see [updating the tests](#updating-the-tests).
+
+### Governance
+
+The [Interop 2023 team](https://github.com/orgs/web-platform-tests/teams/interop-2023) will manage the overall Interop 2023 effort. The team will be formed with the same initial membership as the [’22 team](https://github.com/orgs/web-platform-tests/teams/interop-2022).
+
+The team will meet regularly, with agenda issues created in the repo at least 48 hours in advance, and meeting minutes posted in the issue.
+
+The team makes decisions based on consensus, which is defined as support from at least two participating organizations and no objections. Additionally, during the [proposal selection](#proposal-selection) process, the positions of implementers in each browser engine are used to determine whether a proposal will be accepted, to ensure that the proposal is on track to interoperability across browser engines.
```

Comment:
Can we be consistent about whether we're using "organisations" or "implementers in each browser engine"? These aren't the same. Can we not simply rely on objections for the latter part?

Reply:
My intention was to limit the special status of browser engines and implementers only to the positions process. Do you think we should use one definition of consensus everywhere, and if so what should it be?

Some specific cases to consider, whether in the positions process or other decisions:
- Google and Microsoft support and everyone else is neutral.
- Bocoup and Igalia support and everyone else is neutral. Consider both the case where there is commitment to implement and not.
- Apple, Google, Microsoft and Mozilla support but another newly joined organization objects.

I hope dearly that we won't have to explore the limits of our process and think that in practice we'll have a conversation and resolve things without falling back to the formal definition of consensus. But in particular for the positions process I think we need to be clear, and that what we did last time worked out pretty well.

> Can we not simply rely on objections for the latter part?

Do you mean that decisions require support&times;2 and no objections, and that's it?

Reply:
> My intention was to limit the special status of browser engines and implementers only to the positions process. Do you think we should use one definition of consensus everywhere, and if so what should it be?

Yes, and I'll deal with the second clause below…

> Some specific cases to consider, whether in the positions process or other decisions:
> 
> * Google and Microsoft support and everyone else is neutral.

At first glance, that seems unproblematic to approve in general? Like, okay, there's only positive indications from a single browser engine, so this in a sense depends on whether we believe people will actively object to things? On the whole, I think we generally saw that last year (as I think you mentioned, nothing would've changed had we done this last year), at least after much discussion and it being clear how much support each thing had.

> * Bocoup and Igalia support and everyone else is neutral. Consider both the case where there is commitment to implement and not.

Again, depends on whether we expect people to actually object?

> * Apple, Google, Microsoft and Mozilla support but another newly joined organization objects.

While this might be viewed as a harder question, is this actually a harder question than Bocoup objecting (given AFAIK they do little in the way of implementation work)?

The concern in a sense is anyone acting maliciously, I suppose.

> I hope dearly that we won't have to explore the limits of our process and think that in practice we'll have a conversation and resolve things without falling back to the formal definition of consensus. But in particular for the positions process I think we need to be clear, and that what we did last time worked out pretty well.
> 
> > Can we not simply rely on objections for the latter part?
> 
> Do you mean that decisions require support×2 and no objections, and that's it?

No, I meant just removing the second sentence.

Reply:
I've pushed a change to just use support×2 and no opposition everywhere. That did remove the second sentence referenced here. PTAL?